### ISSUE TITLE
Ignore individual issues per file

### ISSUE THREAD

**Issue by Subcide** [infusionsoft/sleuth-figma-plugin#2]
Great plugin! Any chance of being able to ignore individual issues (and remember them locally) in a way that they don't display again, and don't contribute to the percentages?  Would allow us to get a more realistic coverage % for our files, some of which include screenshots, images, or stuff outside the design system such as OS-level UI in some of our mockups which won't use our design system styles.

**Comment by JeromeBenoitDoctolib** [comment]
Yep, I'm thinking like taking only in account some pages of a figma file, flagged with an indicator in the name of the page (like a ✅, or a string), meaning only the Hi-Fi layouts, and not the crap around, like benchmarks and dirty wireframes. Thanks!
